packages/react/src/profile/infrastructure/ProfileCacheManager.ts

```diff
@@ -49,6 +50,33 @@ export class ProfileCacheManager implements IProfileCacheManager {
     });
   }
 
+  update(
+    profileId: ProfileId,
+    updateFn: <TProfile extends Profile>(current: TProfile) => TProfile,
+  ) {
+    this.client.cache.updateQuery<ProfileData, ProfileVariables>(
```

Comment:
@cesarenaldi I had to introduce cache update in ProfileCacheManager to optimistically flip the new peerToPeerRecommendedByMe flag, but it doesn't work (again) :(
the apollo error says "Missing field 'result' while writing result {}" what could it be?

Reply:
replaced with `updateFragment`